app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -1,382 +1,351 @@
+<string name="progress_ledger_opentx">处理数学中</string>
+<string name="progress_ledger_mlsag">哈希中</string>
+<string name="open_wallet_ledger_missing">请（重新）连接Ledger设备</string>
+<string name="accounts_progress_new">账户创建中</string>
+<string name="toast_ledger_attached">%1$s 已附加</string>
+<string name="toast_ledger_detached">%1$s 已从附加中删除</string>
+<string name="progress_nfc_write">标签写入中</string>
+<string name="nfc_write_failed">标签写入失败！</string>
+<string name="nfc_write_successful">标签写入成功</string>
+<string name="nfc_tag_unsupported">标签不支持NDEF!</string>
+<string name="nfc_tag_size">标签提供%1$d字节，但我们需要%2$d!</string>
+<string name="nfc_tag_read_undef">我不了解标签！</string>
+<string name="nfc_tag_read_what">我不知道你想要什么！</string>
+<string name="nfc_tag_read_success">标签读取成功</string>
+<string name="nfc_tag_tap">NFC可用！</string>
+<string name="unknown_amount" translatable="false">\?\?\?</string>
+<string name="menu_info">显示私密信息</string>
+<string name="menu_streetmode">公共场合模式</string>
+<string name="node_wrong_net" translatable="false">只扫描主网的节点！</string>
+<string name="info_nodes_enabled">Node-o-matiC已可用，点此了解更多.</string>
+<string name="node_height">同步的最新区块: %1$s</string>
+<string name="label_nodes">节点</string>
+<string name="node_name_hint">节点名 (选填)</string>
+<string name="node_address_hint">主机名</string>
+<string name="node_port_hint">端口</string>
+<string name="node_user_hint">账户名 (选填)</string>
+<string name="node_pass_hint">密码 (选填)</string>
+<string name="node_host_unresolved">无法解析主机</string>
+<string name="node_host_empty">我们需要该信息！</string>
+<string name="node_port_numeric">必须为数字</string>
+<string name="node_port_range">必须为1&#8211;65535</string>
+<string name="node_fab_add">添加节点</string>
+<string name="node_refresh_hint">点此刷新！</string>
+<string name="node_test_error">连接发生错误 %1$d</string>
+<string name="node_general_error">连接发生错误</string>
+<string name="node_auth_error">授权失败</string>
+<string name="node_result_label">测试结果：</string>
+<string name="node_result">高度: %1$s (v%2$d), Ping: %3$.0fms, IP: %4$s</string>
+<string name="node_testing">测试IP: %1$s &#8230;</string>
+<string name="node_refresh_wait">请等待扫描工作完成</string>
+<string name="node_create_hint">点此选择/添加节点</string>
+<string name="node_pull_hint">手动添加节点或下拉开始扫描</string>
+<string name="node_scanning">网络扫描中&#8230;</string>
+<string name="node_nobookmark">已自动将最佳%1$d节点加入书签</string>
+<string name="label_test">测试</string><!--note: as in "Test a network connection"-->
+<string name="send_address_resolve_bip70">解析支付URI &#8230;</string>
+<string name="send_address_not_bip70">无法解析支付URI</string>
+<string name="send_address_bip70">支付URI已解析 &#x2714;</string>
+<string name="send_address_hint">收款人</string>
+<string name="xmrto_error_001">XMR.TO离线-请稍后重试</string>
+<string name="xmrto_error_004">比特币金额超出限制</string>
+<string name="xmrto_error_010">无效或过期的支付URL</string>
+<string name="xmrto_error_012">当前请求数过多</string>
+<string name="xmrto_error" translatable="false">XMR.TO服务发生错误</string>
+<string name="street_sweep_amount">所有东西！</string>
```

Comment:
Please check if these translation are accurate. This string means all the money in this account.